src/Classes/ServiceAPI/MyRadio_Season.php

```diff
@@ -400,7 +395,7 @@ public function getEditForm()
                 $this->getID(),
                 [
                     'description' => $this->getMeta('description'),
-                    'tags' => implode(' ', $this->getMeta('tag')),
+                    'tags' => implode(',', $this->getMeta('tag')),
```

Comment:
`', '` ?

Reply:
True, however given the entire edit season page doesn't work anyway...